docs/naivebayes.md

```diff
@@ -0,0 +1,108 @@
+Naive Bayes Classifiers
+=======================
+
+The Naive Bayes classifiers is a simple probabilistic classification model based on Bayes Theorem. It is a simple, yet powerful classification model. Since Naive Bayes classifiers simply classifies sets of data by which class has the highest likelihood, Naive Bayes classifiers can use any dsitribution or model that outputs a probability.
```

Comment:
Maybe also use the term "component" to mean distribution + model, such as "Naive Bayes classifiers can use any distribution or model that outputs a probability as one of its components." Because you can have one component be a HMM and one be a NormalDistribution, theoretically.